Tests/BootCamp.Chapter.Tests/ConsoleTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using BootCamp.Chapter.Tests.Utils;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class ConsoleTests: IDisposable
+    {
+        /// <summary>
+        /// Disposes console output file and returns its' contents.
+        /// </summary>
+        protected string ConsoleOutput
+        {
+            get
+            {
+                if (_consoleOutput == null)
+                {
+                    throw new ArgumentNullException(nameof(_consoleOutput));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The parameter name '_consoleOutput' is not declared in the argument list.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4884711)